MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -406,12 +401,11 @@ protected virtual Stream OpenStream(string assetName)
 			{
 				throw new ContentLoadException("The content file was not found.", fileNotFound);
 			}
-#if !WINDOWS_UAP
+
```

Comment:
Should the 'W' in line 45 be moved to legacy identifiers? Also the 'M' there :)

Reply:
Thanks for the review, I did remove all uap "stuff" but will check when I'm back in the UK.